src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RENT_PRISE = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const TOTAL_PRICE = DAILY_RENT_PRISE * days;
```

Comment:
```suggestion
  const totalPrice = DAILY_RENT_PRISE * days;
```

If value is calculated we name it in `camelCase`